docs/how-to/test-update-system-locally.md

```diff
@@ -0,0 +1,39 @@
+# Testing update system locally
+
+This instruction covers how to test software update locally:
+
+  1. Prepare and propose software update
+      a. Prepare alt configuration with changed application version constant
+      b. Propose updated with script that simply replaces one configuration file (in setup) with other
+  2. Launch wallet node with launcher, Daedalus (or Daedalus emulator)
```

Comment:
What's “Daedalus emulator”? It's just a nitpick but I'd like this sentence to be clearer

Reply:
renamed to _script that serves as Daedalus emulator_